packages/nc-gui-v2/components/dashboard/TreeView.vue

```diff
@@ -352,7 +352,7 @@ function openTableCreateDialog() {
                           </div>
                         </a-menu-item>
 
-                        <a-menu-item v-if="isUIAllowed('table-delete')" @click="() => $e('c:table:delete') && deleteTable(table)">
```

Comment:
If this gets removed there's no telemetry event sent out as we can't bind the directive onto `a-menu-item`

Reply:
It's already emitting event from corresponding function in composable

Reply:
ah okay I missed that 👍